<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMergeConfigurations</class>
 <widget class="QDialog" name="DlgMergeConfigurations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="VLayoutSource">
           <item>
            <widget class="QLabel" name="LabelSource">
             <property name="text">
              <string>Available configurations</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="ListWidgetSource"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="VLayoutDestination">
           <item>
            <widget class="QLabel" name="LabelDestination">
             <property name="text">
              <string>Destination</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="ListWidgetDestination"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Hover over the items to see the details of their configuration</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VLayoutAction">
       <item>
        <spacer name="VSpacerAction1">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonMerge">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Merge</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonOverwrite">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Overwrite</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="VSpacerAction2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutClose">
     <item>
      <spacer name="HSpacerClose">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
